trashtalk/templates/users/new.html

```diff
@@ -1,4 +0,0 @@
-{% block content %}
```

Comment:
We want to keep this. Having a static page that serves this purpose is a good fallback for users that have issues with modals.